content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -75,11 +75,8 @@ for provisioning PVs. This field must be specified.
 
 | Volume Plugin        | Internal Provisioner |            Config Example             |
 | :------------------- | :------------------: | :-----------------------------------: |
-| AWSElasticBlockStore |       &#x2713;       |          [AWS EBS](#aws-ebs)          |
 | AzureFile            |       &#x2713;       |       [Azure File](#azure-file)       |
-| AzureDisk            |       &#x2713;       |       [Azure Disk](#azure-disk)       |
 | CephFS               |          -           |                   -                   |
-| Cinder               |       &#x2713;       | [OpenStack Cinder](#openstack-cinder) |
 | FC                   |          -           |                   -                   |
 | FlexVolume           |          -           |                   -                   |
 | GCEPersistentDisk    |       &#x2713;       |           [GCE PD](#gce-pd)           |
```

Comment:
Remove

Reply:
According to https://github.com/kubernetes/website/pull/42302#discussion_r1282723644, GCE PD is still in Kubernetes 1.27